tests/extensions/aria_extension_tosca/simple_v1_0/data.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
```

Comment:
why is it called data.py and not constants.py?

Reply:
Because 1) Python doesn't have a "constants" feature, and 2) even if it did, that would be an implementation trait, and these data points would still work as data points if they weren't constants. I don't see why you would want to name the file for a technical aspect of its content. It would be like separating this file into `floats.py` and `ints.py` -- that doesn't help understanding usage.